hermes/validator/question_generator.py

```
@@ -78,7 +79,8 @@ async def generate_question(
             round_id: int = 0,
             weight_a: int = 70,
             weight_b: int = 30,
-            project_frequency: dict[str, int] = {}
+            project_frequency: dict[str, int] = {},
```

Ruff (0.15.10)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)